src/lib/isStrongPassword.js

```diff
@@ -4,7 +4,7 @@ import assertString from './util/assertString';
 const upperCaseRegex = /^[A-Z]$/;
 const lowerCaseRegex = /^[a-z]$/;
 const numberRegex = /^[0-9]$/;
-const symbolRegex = /^[-#!$%^&*()_+|~=`{}\[\]:";'<>?,.\/ ]$/;
+const symbolRegex = /^[-#!$@%^&*()_+|~=`{}\[\]:";'<>?,.\/ ]$/;
```

Comment:
Should we do `/^[\W_ ]$/` instead?

Reply:
I think it's better to hardcode specific characters.

Reply:
`\w` also includes underscores.

Reply:
If that's the case then we should include in the documentation that this function is restricted to ASCII characters only and symbols such as `º` or `§` will not work, which will be quite a surprise for some locales that have these symbols in their keyboards.

It would work with `/^[\W_ ]$/` though.

Reply:
Hmm, that's true. I don't know if people use these "weird" characters in their password? Personally I've never seen someone do that but I dunno. Maybe it is better to do that other regex